src/components/accordion/Accordion.jsx

```diff
@@ -67,6 +67,7 @@ type ContextType = {
   dispatch: (action: ActionType) => void,
   reduceMotion: boolean,
   onItemSelect: (id: string, value: boolean) => void,
+  isControlled: boolean,
```

Comment:
I think `isControlled` is unnecessary here in context. Maybe we can pass list of disabled accordion items controlled by accordion? Item could be marked as disabled when for instance `allowMultiple` is false and item is already expanded?

Reply:
`allowMultiple` is not working in controlled Accordion, in that case only one accordion item can be expanded and then must be disabled (it can be closed only by expanding some other item).

Reply:
You are right, controlled Accordion doesn't care about `allowMultiple`. As we discussed it, If someone wants to control Accordion, he should also control `aria-disable` attributes of accordion items.